solutionbox/inception/datalab_solutions/inception/_cloud.py

```diff
@@ -60,22 +60,21 @@ def preprocess(self, input_csvs, labels_file, output_dir, pipeline_option=None):
 
     opts = beam.pipeline.PipelineOptions(flags=[], **options)
     p = beam.Pipeline('DataflowPipelineRunner', options=opts)
-    _preprocess.configure_pipeline(
-        p, self._checkpoint, input_csvs, labels_file, output_dir, job_name)
+    _preprocess.configure_pipeline(p, self._checkpoint, input_csvs, output_dir, job_name)
     p.run()
 
-  def train(self, labels_file, input_dir, batch_size, max_steps, output_path,
+  def train(self, input_dir, batch_size, max_steps, output_path,
             region, scale_tier):
     """Cloud training with CloudML trainer service."""
 
     import datalab.mlalpha as mlalpha
-    num_classes = len(_util.get_labels(labels_file))
+    labels = _util.get_labels(input_dir)
     job_args = {
       'input_dir': input_dir,
       'output_path': output_path,
       'max_steps': max_steps,
       'batch_size': batch_size,
-      'num_classes': num_classes,
+      'labels': labels,
```

Comment:
We actually can get labels from input_dir, so this is not needed. Thanks for catching this!